client/src/cli/rmmbrAccessToken.ts

```diff
@@ -0,0 +1,7 @@
+import { getAccessTokenPath } from "./accessTokenPath.ts";
```

Comment:
do we need this file? doesn't look like it adds much
if we do then avoid using the service name in the title

Reply:
I do use this function in two files and I don't want to set it up twice... wdyt?